.github/workflows/docker-image.yml

```diff
@@ -0,0 +1,39 @@
+name: Docker Image CI
+
+on:
+  push:
+    branches: ["petanitest"]
```

Comment:
Change branch name to "master" since we only deploy to production when master there are commits to that remote branch